.gitignore

```diff
@@ -1,2 +1,4 @@
 bin/
 *.log
+.idea/
```

Comment:
this change probably belongs on your local system but not as part of the group project